src/components/connection-prompt/ConnectDialogContainer.svelte

```diff
@@ -190,7 +190,8 @@
     onClose={connectionStateNone}
     hasCloseButton={$connectionDialogState.connectionState !==
       ConnectDialogStates.USB_DOWNLOADING}
-    dismissOnClickOutside={false}>
+    closeOnOutsideClick={false}
+    closeOnEscape={false}>
```

Comment:
I think we only want to make `closeOnEscape` true when we're flashing. So same as the `hasCloseButton` above.